sysdump/sysdump.go

```diff
@@ -144,21 +146,19 @@ func Collect(c *k8s.Client, o Options) error {
 			return err
 		}
 		for _, p := range pods.Items {
-			if p.Annotations["kubearmor-policy"] == "enabled" {
-				v, err := c.K8sClientset.CoreV1().Pods(p.Namespace).Get(context.Background(), p.Name, metav1.GetOptions{})
-				if err != nil {
-					return err
-				}
-				if err := writeYaml(path.Join(d, p.Namespace+"-pod-"+p.Name+".yaml"), v); err != nil {
-					return err
-				}
-				e, err := c.K8sClientset.CoreV1().Events(p.Namespace).Search(scheme.Scheme, v)
-				if err != nil {
-					return err
-				}
-				if err := writeYaml(path.Join(d, p.Namespace+"-pod-events-"+p.Name+".yaml"), e); err != nil {
-					return err
-				}
+			v, err := c.K8sClientset.CoreV1().Pods(p.Namespace).Get(context.Background(), p.Name, metav1.GetOptions{})
+			if err != nil {
+				return err
+			}
+			if err := writeYaml(path.Join(d, p.Namespace+"-pod-"+p.Name+".yaml"), v); err != nil {
+				return err
+			}
+			e, err := c.K8sClientset.CoreV1().Events(p.Namespace).Search(scheme.Scheme, v)
+			if err != nil {
+				return err
+			}
+			if err := writeYaml(path.Join(d, p.Namespace+"-pod-events-"+p.Name+".yaml"), e); err != nil {
+				return err
```

Comment:
It would be nice to know which pods do not have the annotations applied. We could figure out the start time, end time etc and try to reason why the annotations are not applied. But without any information, it would be difficult to reason. Just speculating here 🙂